datascience/utils.py

```diff
@@ -41,3 +43,31 @@ def get_production(countries, start_date, end_date, delta):
                               'production': o['production'].values()})
             df = df.append(p)
     return df
+
+# get_exchange
+
+def fetch_exchange(country_code, t):
+    endpoint = 'http://electricitymap.tmrow.co'
+    url = '%s/v1/state' % endpoint
```

Comment:
The `/state` api returns both exchanges and production, so the same argument would apply for `get_production`.
Either we keep one endpoint (`/state`) and pull both exchanges, production and co2, or we make individual endpoints.
Also, the co2 computation requires all exchanges and all productions. When querying production or exchanges however, we can dramatically increase response time by querying only specific exchanges/countries.
Therefore, I suggest to make a specific exchange endpoint, and add a `get_co2` function that will hit `/state` and return all 3 variables for all zones.